src/test/regress/expected/pg14.out

```diff
@@ -32,14 +82,6 @@ NOTICE:  issuing REINDEX (TABLESPACE test_tablespace) INDEX pg14.xxxxx
 DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
 NOTICE:  issuing REINDEX (TABLESPACE test_tablespace) INDEX pg14.xxxxx
 DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
-NOTICE:  issuing REINDEX (TABLESPACE test_tablespace) INDEX pg14.xxxxx
```

Comment:
hmm, why are these not printed anymore?

Reply:
once we fix this, I'm fine to merge

Reply:
This is because shard_count is set to 2, to prevent verbosity.